include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -285,8 +239,9 @@ struct is_temp_buff : ::std::false_type
 };
 
 template <typename _Iter>
-struct is_temp_buff<_Iter, ::std::enable_if_t<!is_sycl_iterator_v<_Iter> && !::std::is_pointer_v<_Iter> &&
-                                              !is_passed_directly_v<_Iter>>> : ::std::true_type
+struct is_temp_buff<_Iter, std::enable_if_t<!is_sycl_iterator_v<_Iter> && !std::is_pointer_v<_Iter> &&
```

Comment:
Should we write here full qualified name for `is_sycl_iterator_v` (include namespace)?

Reply:
yes. fixed.